R/tree_accessors.R

```diff
@@ -2877,33 +2877,70 @@ setMethod("cell_footnotes<-", "ContentRow",
   definition = .cfn_set_helper
 )
 
+# Deprecated methods
 #' @export
 #' @rdname ref_fnotes
 setGeneric("col_fnotes_here", function(obj) standardGeneric("col_fnotes_here"))
 #' @export
+#' @rdname ref_fnotes
+setMethod("col_fnotes_here", "ANY", function(obj) {
+  warning("col_fnotes_here is deprecated since {rtables} version 0.6.5.9011. Please use col_footnotes instead.")
+  col_footnotes(obj)
+})
+#' @export
+#' @rdname ref_fnotes
+setGeneric("col_fnotes_here<-", function(obj, value) standardGeneric("col_fnotes_here<-"))
+#' @export
+#' @rdname int_methods
+setMethod("col_fnotes_here<-", "ANY", function(obj, value) {
+  warning("col_fnotes_here<- is deprecated since {rtables} version 0.6.5.9011. Please use col_footnotes<- instead.")
+  col_footnotes(obj) <- value
```

Comment:
@edelarua is this a valid deprecation cycle, what do you think?

Reply:
Looks good to me!